README.md

```diff
@@ -1 +1,31 @@
-# flexy-thingworx-connector
\ No newline at end of file
+# Flexy Thingworx Connector
+
+The Flexy Thingworx Connector provides a connector-based solution development kit for linking the Ewon Flexy to a Thingworx instance.
+
+There are two components that make up the Flexy Thingworx Connector, a Thingworx service and a Flexy Java app.
+
+## Table of Contents
+1. [Thingworx Service Component](#thingworx-service-component)
+2. [Flexy Java App Component](#flexy-java-app-component)
+   1. [Installation](#installation)
+   2. [Source Code](#source-code)
+      1. [Development Environment](#development-environment)
+
+## Thingworx Service Component
+The Thingworx service component must be installed for both data path configurations (Direct and Talk2M).
```

Comment:
I dont think I have seen any documentation that would specify the difference between direct and talk2m at this point. I would think these would need to be defined before this point in the readme.

Reply:
added